wp-content/themes/pub/wporg-learn-2024/inc/block-hooks.php

```diff
@@ -15,6 +15,8 @@
 
 /**
  * Update header template based on current query.
+ * Since the search results for courses and lessons still use their respective archive templates,
+ * we need to update the header template part to display the correct title.
```

Comment:
Documentation improvements like this could be committed right to trunk IMO, to avoid extra changes in PRs like this. No change requested for this PR, just a suggestion for the future 🙂

Reply:
Noted, thanks!